__tests__/runner_test.ml

```diff
@@ -32,49 +32,91 @@ let () =
     Js.String.length input == 3);
   testAll "testAll - tuples" [("foo", 3); ("barbaz", 6); ("bananas!", 8)] (fun (input, output) ->
     Js.String.length input == output);
-  
+
   describe "describe" (fun () ->
     test "some aspect" (fun () ->
       true)
   );
-  
-  describe "beforeAll" (fun () -> 
+
+  describe "beforeAll" (fun () ->
     let x = ref 0 in
-    
```

Comment:
This isn't a very big deal in this case, but in general it's a good idea to turn off automatic whitespace removal when submitting PRs to other projects since it creates a lot diff noise.

Reply:
Sorry about that. reverted.